source/_integrations/airzone_cloud.markdown

```diff
@@ -0,0 +1,39 @@
+---
+title: Airzone Cloud
+description: Instructions on how to integrate Airzone Cloud within Home Assistant.
+ha_release: 2023.6
+ha_category:
+  - Sensor
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_domain: airzone_cloud
+ha_platforms:
+  - sensor
+ha_codeowners:
+  - '@Noltari'
+ha_integration_type: integration
+---
+
+This integration interacts with the Cloud API of [Airzone HVAC zoning systems](https://www.airzone.es/en/).
+
+A typical Airzone device has a *parent zone* (Master Thermostat) per HVAC system, which is the only zone where the HVAC mode can be changed. The rest are *child zones* which can only enable or disable the HVAC and adjust the desired temperature on that specific zone.
```

Comment:
I'm not familiar with Airzone and therefore struggled a bit when reading this. Some questions:
1. What are these zones? Could you maybe write an introductory sentence on what a zone is in this context? Is it something like my kitchen or living room?
1. Instead of using the master/slave terminology (master thermostat), would any of these [alternative terms](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/m/master-slave) work? 
1. What is a HVAC mode? What are possible values?

Reply:
> I'm not familiar with Airzone and therefore struggled a bit when reading this. Some questions:
> 
> 1. What are these zones? Could you maybe write an introductory sentence on what a zone is in this context? Is it something like my kitchen or living room?
> 2. Instead of using the master/slave terminology (master thermostat), would any of these [alternative terms](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/m/master-slave) work?
> 3. What is a HVAC mode? What are possible values?

1. Zones means rooms with Airzone Thermostats in Airzone terminology. There's more information in these links (from the official website linked on the top):
- https://www.airzone.es/en/comfort/
- https://doc.airzone.es/AZEU/Marketing/The_New_Era_Of_Air_Conditioning_Airzone.pdf
2. Airzone uses Master to refer to the parent thermostat, so I prefer to keep that single "parent zone" == "Master Thermostat" relation to make it more clear to the users reading it.
3. See "hvac_mode" in https://www.home-assistant.io/integrations/climate/